main/src/ca/uwaterloo/flix/language/ast/SyntaxTree.scala

```diff
@@ -196,6 +196,8 @@ object SyntaxTree {
 
       case object ExtensibleMatch extends Expr
 
+      case object ExtensibleMatchRuleFragment extends Expr
```

Comment:
In my code I named these things ExtensibleFoo.

But let us rename them to ExtFoo as you go. Then we know that if it is ExtFoo then we are done with it-- so to speak